docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,81 @@
+# Tutorial: Deploy the Penpot charm
+
+## What you'll do
+This tutorial will walk you through deploying the penpot charm; you will:
+1. Deploy and configure the penpot charm's dependencies (PostgreSQL, Redis, s3, etc.)
+2. Deploy Penpot and integrate it with its dependencies
+3. Create a user
+4. Login to the web interface for your deployed Penpot instance
+
+## Prerequisites
+* A kubernetes cluster.
+* A host machine with juju version 3.4 or above.
```

Comment:
Nitpick: Capitalise Juju (here and other places outside of code blocks)

Reply:
Sounds good, will do.